packages/editor-ui/src/components/RunDataSchema.vue

```diff
@@ -71,7 +72,7 @@ const onDragEnd = (el: HTMLElement) => {
 
 <template>
 	<div :class="$style.schemaWrapper">
-		<div v-if="isDataEmpty" />
+		<n8n-alert v-if="isDataEmpty" :background="false" :description="i18n.baseText('dataMapping.schemaView.emptyData')" />
```

Comment:
would not info tip be better here based on max's designs? https://n8n-io.github.io/n8n-storybook/?path=/story/atoms-infotip--note

Reply:
also since you have implemented this, Max wanted this for for table view as well

Reply:
In this particular case probably yes. I'll switch them, but maybe we should merge these 2 components (like we did with the buttons) because `alert` is much more configurable.